src/shared/constants/validation.ts

```diff
@@ -1 +1,8 @@
-export const CORE_PLUGINS = ['open-policy-agent', 'kubernetes-schema', 'yaml-syntax', 'resource-links'];
+export const CORE_PLUGINS = [
```

Comment:
do we really have to hard-code this list? I'm sure these will change over time in core - and we might forget keeping it in sync..

Reply:
One solution would be to have this list directly in the validation and use it from there. Right now we need this kind of list to load the initial validation with the core plugins...